library/error.c

```diff
@@ -404,7 +404,7 @@ void mbedtls_strerror( int ret, char *buf, size_t buflen )
         // END generated code
 
         if( strlen( buf ) == 0 )
-            mbedtls_snprintf( buf, buflen, "UNKNOWN ERROR CODE (%04X)", use_ret );
+            mbedtls_snprintf( buf, buflen, "UNKNOWN ERROR CODE (%04X)", (unsigned int)use_ret );
```

Comment:
`library/error.c` is automatically generated. Please edit `scripts/data_files/error.fmt` instead, then run `scripts/generate_errors.pl`.